src/matchesWhereQuery.es

```diff
@@ -50,9 +50,13 @@ module.exports = function( object, whereQuery ) {
 				break;
 			case 'contains':
 				if( ! objectAttribute ) return false;
```

Comment:
I think this is not needed, since it would be caugh in the latest `else`, right?

Reply:
This check is needed as `objectAttribute` is not the same as `queryAttribute`, so we need to check there's something in it. Fixed conflicts.